keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -342,11 +346,21 @@ ClusterTrustBundle objects may optionally be associated with a signer using the
 additional admission control and access control for the ClusterTrustBundle
 object, as detailed in the [Access Control](#access-control) section.
 
-While multiple ClusterTrustBundle objects may be associated with a signer, each
-is a completely independent expression of the roots of trust for the signer.  No
-effort is made to unify trust anchors from different ClusterTrustBundle objects.
-The primary use case for multiple ClusterTrustBundle objects for a single signer
-is [canarying](#canarying-changes-to-a-clustertrustbundle).
+ClusterTrustBundles that are associated with a signer have a required naming
+prefix:  take the signer name, replace slashes (`/`) with colons (`:`), and
+append a colon at the end.  This must then be followed by an additional
+freely-chosen name that does not contain colons.  For example, valid
+ClusterTrustBundle names for the signer `example.com/mysigner` would be
+`example.com:mysigner:foo` and `example.com:mysigner:live`.
+
+ClusterTrustBundles that are not associated with a signer may be freely named, as long as their name does not contain a colon.
+
+Multiple ClusterTrustBundle objects may be associated with a single signer.
+While each object is independent at the API level, consumers (mostly Kubelet)
+will select trust anchors by a combination of field selector on signer name, and
+a label selector.  Signer controllers should follow the convention of making the
+label selector `kubernetes.io/cluster-trust-bundle-version=live` correspond to a
+meaningful set of trust anchors.  In general, users are expected to read the documentation for their signer controller implementation in order to determine which label selectors to use for their needs, including [canarying](#canarying-changes-to-a-clustertrustbundle).
```

Comment:
I don't recall this convention being part of the API?

Reply:
It is not part of the API.  It's a suggested convention.  I think it's premature to include it now, we should wait until we export the roots of some built-in signers.  Removing it.

Reply:
actually, it looks like this is threaded through all the KEP examples.  I've converted "should" to "may"